control-services/images/semaphore/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ansiblesemaphore/semaphore:v2.8.21
+FROM ansiblesemaphore/semaphore:local
```

Comment:
Can you change the docker image name from `local` to something more descriptive?  Possibly `dynamicVariable` or `homebrew`.  Or create an enhancement issue for that change if its a major hurdle?

Reply:
I think something describing the feature is best.  Changing from `latest` to `local` doesn't feel more intuitive to me